youtube_dl/extractor/polskieradio.py

```diff
@@ -15,6 +16,63 @@
 )
 
 
+class PolskieRadioProgrammeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?polskieradio\.pl/(?P<id>\d+/\d+)(,.*)?(\?.*)?$'
```

Comment:
Do you really think that's a cleaner solution? It seems to me that there should be a single canonical way to define which extractor handles which URLs, as is with the `_VALID_URL` constant, and I think the assumption that no two extractors collide in that regard is fundamental.